### PR TITLE
[zh] Translate reference/access-authn-authz/authentication.md

### PR DESCRIPTION
**Pull request by tengqm**
closes: #22943

### CODE REVIEW

**Review by markthink — COMMENTED** [review]
/lgtm
/approve

File: content/zh/docs/reference/access-authn-authz/authentication.md [hunk 1]
```diff
@@ -0,0 +1,1549 @@
...
+ - at least one other method for user authentication.
+-->
+所有（属性）值对于身份认证系统而言都是不透明的，只有被
+[鉴权组件](/zh/docs/reference/access-authn-authz/authorization/)
+解释过之后才有意义。
+
+你可以同时启用多种身份认证方法，并且你通常会至少使用两种方法：
+
+- 针对服务账号使用服务账号令牌
+- 至少另外一种方法对认证用户的身份
```
[the remark this answers is not in the document]
Comment (tengqm) [reply content/zh/docs/reference/access-authn-authz/authentication.md:139]: fixed.

File: content/zh/docs/reference/access-authn-authz/authentication.md [hunk 2]
```diff
@@ -0,0 +1,1549 @@
...
+<!--
+To allow for streamlined bootstrapping for new clusters, Kubernetes includes a
+dynamically-managed Bearer token type called a *Bootstrap Token*. These tokens
+are stored as Secrets in the `kube-system` namespace, where they can be
+dynamically managed and created. Controller Manager contains a TokenCleaner
+controller that deletes bootstrap tokens as they expire.
+-->
+为了支持平滑地启动引导新的集群，Kubernetes 包含了一种动态管理的持有者令牌类型，
+称作 *启动引导令牌（Bootstrap Token）*。
+这些令牌一 Secret 的形式保存在 `kube-system` 名字空间中，可以被动态管理和创建。
```
[the remark this answers is not in the document]
Comment (tengqm) [reply content/zh/docs/reference/access-authn-authz/authentication.md:272]: fixed.

File: content/zh/docs/reference/access-authn-authz/authentication.md [hunk 3]
```diff
@@ -0,0 +1,1549 @@
...
+这里的用户名和组名可以用来（并且已经被 `kubeadm` 用来）构造合适的鉴权
+策略，以完成启动引导新集群的工作。
+
+<!--
+Please see [Bootstrap Tokens](/docs/reference/access-authn-authz/bootstrap-tokens/) for in depth
+documentation on the Bootstrap Token authenticator and controllers along with
+how to manage these tokens with `kubeadm`.
+-->
+请参阅[启动引导令牌](/zh/docs/reference/access-authn-authz/bootstrap-tokens/)
+以了解关于启动引导令牌身份认证组件与控制器得更深入的信息，以及如何使用
```
[the remark this answers is not in the document]
Comment (tengqm) [reply content/zh/docs/reference/access-authn-authz/authentication.md:321]: right.

File: content/zh/docs/reference/access-authn-authz/authentication.md [hunk 4]
```diff
@@ -0,0 +1,1549 @@
...
+-->
+### 服务账号令牌   {#service-account-tokens}
+
+服务账号通常由 API 服务器自动创建并通过 `ServiceAccount` 
+[准入控制器](/zh/docs/reference/access-authn-authz/admission-controllers/)
+关联到集群中运行的 Pod 上。
+持有者令牌会挂载到 Pod 中可预知的为之，允许集群内进程与 API 服务器通信。
+服务账号也可以使用 Pod 规约的 `serviceAccountName` 字段显式地关联到 Pod 上。
+
+<!--
```
[the remark this answers is not in the document]
Comment (tengqm) [reply content/zh/docs/reference/access-authn-authz/authentication.md:381]: 漏翻了一段。已补齐。

**Comment by tengqm** [comment]
@tanjunchen Thanks for the careful review. All feedbacks addressed.

File: content/zh/docs/reference/access-authn-authz/authentication.md [hunk 5]
```diff
@@ -0,0 +1,1564 @@
...
+Prior to 1.11.3 (and 1.10.7, 1.9.11), the extra key could only contain characters which were [legal in HTTP header labels](https://tools.ietf.org/html/rfc7230#section-3.2.6).
+For example, with this configuration:
+-->
+{{< note >}}
+在 1.13.3 版本之前（包括 1.10.7、1.9.11），附加字段的键名只能包含
+[HTTP 头部标签的合法字符](https://tools.ietf.org/html/rfc7230#section-3.2.6)。
+{{< /note >}}
+
+例如，使用下面的配置：
+
```
Comment (tanjunchen) [inline content/zh/docs/reference/access-authn-authz/authentication.md:1021]: review here. line 1020

File: content/zh/docs/reference/access-authn-authz/authentication.md [hunk 6]
```diff
@@ -0,0 +1,1564 @@
...
+    user: my-user
+current-context: my-cluster
+```
+
+<!--
+Relative command paths are interpreted as relative to the directory of the config file. If
+KUBECONFIG is set to `/home/jane/kubeconfig` and the exec command is `./bin/example-client-go-exec-plugin`,
+the binary `/home/jane/bin/example-client-go-exec-plugin` is executed.
+-->
+配置文件的的相对路径表达都作为与配置文件的相对路径来解析。
```
Comment (tanjunchen) [inline content/zh/docs/reference/access-authn-authz/authentication.md:1458]: 有点不通顺

File: content/zh/docs/reference/access-authn-authz/authentication.md [hunk 7]
```diff
@@ -0,0 +1,1564 @@
...
+
+To use bearer token credentials, the plugin returns a token in the status of the `ExecCredential`.
+-->
+### 输出和输出格式   {#input-and-output-formats}
+
+所执行的命令会在 `stdout` 打印 `ExecCredential` 对象。
+`k8s.io/client-go` 使用 `status` 中返回的凭据信息向 Kubernetes API 服务器
+执行身份认证。
+
+在交互式回话中运行时，`stdin` 是直接暴露给插件使用的。
```
Comment (tanjunchen) [inline content/zh/docs/reference/access-authn-authz/authentication.md:1490]: 在交互式回话中运行时？

**Pull request merged**